mantle/kola/tests/ostree/unlock.go

```diff
@@ -44,8 +44,8 @@ func init() {
 }
 
 var (
-	rpmUrl  string = "https://raw.githubusercontent.com/projectatomic/atomic-host-tests/master/rpm/aht-dummy-1.0-1.noarch.rpm"
-	rpmName string = "aht-dummy"
+	rpmUrl  string = "https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-rawhide.noarch.rpm"
```

Comment:
You can't just use the rawhide RPM on F42 (testing-devel) so you'd have to conditionalize everything. I'd say at this point let's just try to build the dummy rpm again in a new dedicated repo for that purpose.